runtime/interpreter/interpreter.go

```diff
@@ -2571,6 +2571,36 @@ var converterDeclarations = []valueConverterDeclaration{
 	},
 }
 
+func lookupInterface(interpreter *Interpreter, typeID string) (*sema.InterfaceType, bool) {
```

Comment:
Maybe return the errors instead of ignoring them and returning a boolean. That way callers of the function can use the error if needed, or ignore them / translate them to a boolean.'

Same for `lookupComposite`